mycroft/audio/service.py

```diff
@@ -218,6 +218,16 @@ def handle_queue_audio(self, message):
         listen = message.data.get("listen", False)
         TTS.queue.put((audio_ext, str(audio_file), viseme, ident, listen))
 
+    def handle_get_languages_tts(self, message):
+        """
+        Handle a request for supported TTS languages
+        :param message: neon.get_languages_tts request
+        """
+        tts_langs = self.tts.available_languages or \
+            [self.config.get('language', {}).get('user') or 'en-us']
+        LOG.info(f"Got tts_langs: {tts_langs}")
+        self.bus.emit(message.response({'tts_langs': tts_langs}))
```

Comment:
can we unify this data field as "langs" and change only the message type between services?

also update the log to DEBUG

Reply:
makes sense; that will simplify parsing on the other end as well